src/components/screens/monitor/network/overview.js

```diff
@@ -0,0 +1,146 @@
+// istanbul ignore file
+import React from 'react';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxEmptyState from '../../../toolbox/box/emptyState';
+import { DoughnutChart } from '../../../toolbox/charts';
+import styles from './overview.css';
+
+const createOthers = (data) => {
+  const list = {};
+  const keys = Object.keys(data);
+  let Others = 0;
+  const sortedKeys = Object.entries(data)
+    .sort((a, b) => a[1] - b[1])
+    .reverse()
+    .map(item => item[0]);
+  sortedKeys.forEach((item, index) => {
+    if (index < 3) {
+      list[item] = data[item];
+    } else {
+      Others = data[item] + Others;
+    }
+  });
+  return keys.length <= 2
+    ? data
+    : { ...list, Others };
+};
+const Overview = ({
+  networkStatus,
+  t,
+}) => {
+  const { basic, coreVer, height } = networkStatus;
+  const versionsDistribution = coreVer ? createOthers(coreVer) : null;
+  const heightDistribution = height ? createOthers(height) : null;
+  return (
+    <Box>
+      <BoxHeader>
+        <h1>{t('Statistics')}</h1>
+      </BoxHeader>
+      <div className={`${styles.container} ${styles.overview}`}>
+        <div className={styles.column}>
+          {
+            versionsDistribution
+              ? (
+                <div className={styles.chartBox}>
+                  <h2 className={styles.title}>{t('Height distribution')}</h2>
```

Comment:
This is the title of height distribution but It's above the version distribution chart.